### ISSUE TITLE
Sdk size

### ISSUE THREAD

**Issue by JenishKananiLime** [mapbox/mapbox-maps-ios#1389]
## Environment

- Xcode version:
- iOS version:
- Devices affected:
- Maps SDK Version:

## Observed behavior and steps to reproduce
On mapbox documentation its says the sdk takes 3-4 mb of app size, but the sdk in total is 11 mb, including size of mapbox common frame work, is this a known bug?

**Comment by ZiZasaurus** [comment]
Hi there, can you please link the documentation where it mentions that the sdk takes 3-4 mb of app size?

**Comment by JenishKananiLime** [comment]
https://docs.mapbox.com/help/troubleshooting/ios-framework-size/

**Comment by ZiZasaurus** [comment]
@JenishKananiLime, our binary size baselines can be found [here](https://github.com/mapbox/mapbox-maps-ios/blob/main/baselines/binary-size/MapboxMaps.json), where the `MapboxMaps_iphoneos_arm64` is just a little over 2 mb.

**Comment by JenishKananiLime** [comment]
yes but is the overall sdk size bigger? then 2mb

**Comment by ZiZasaurus** [comment]
The numbers that are mentioned in the documentation you are referencing are based on app thinning. `MapboxMaps_iphoneos_arm64` is what actually would be on the real device inside your application.

**Comment by JenishKananiLime** [comment]
Yes but we did a app thining report and the mapbox common framework is a very big library. More then 5mb

**Comment by JenishKananiLime** [comment]
@OdNairy will add next week

**Comment by JenishKananiLime** [comment]
App Thinning Size Report for All Variants of LimebikeRelease

Variant: Lime-E7091CB4-80A8-4676-BCA3-E2B24F360DC4.ipa
Supported variant descriptors: [device: iPhone14,3, os-version: 15.0], [device: iPhone10,3, os-version: 15.0], [device: iPhone13,2, os-version: 15.0], [device: iPhone11,4, os-version: 15.0], [device: iPhone14,5, os-version: 15.0], [device: iPhone13,1, os-version: 15.0], [device: iPhone10,5, os-version: 15.0], [device: iPhone14,4, os-version: 15.0], [device: iPhone12,3, os-version: 15.0], [device: iPhone13,4, os-version: 15.0], [device: iPhone14,2, os-version: 15.0], [device: iPhone10,6, os-version: 15.0], [device: iPhone11,6, os-version: 15.0], [device: iPhone11,2, os-version: 15.0], [device: iPhone9,2, os-version: 15.0], [device: iPhone10,2, os-version: 15.0], [device: iPhone13,3, os-version: 15.0], [device: iPhone9,4, os-version: 15.0], and [device: iPhone12,5, os-version: 15.0]
App + On Demand Resources size: 57.5 MB compressed, 128 MB uncompressed
App size: 120.9 MB uncompressed
On Demand Resources size: 7.2 MB uncompressed

**Comment by JenishKananiLime** [comment]
<img width="241" alt="Screen Shot 2022-07-01 at 6 14 55 PM" src="https://user-images.githubusercontent.com/79174176/176973408-78bb8e42-453a-4402-9df9-2a718b80f32c.png">
<img width="288" alt="Screen Shot 2022-07-01 at 6 15 01 PM" src="https://user-images.githubusercontent.com/79174176/176973416-67593ac6-214f-4c08-b89f-3c74cc4f7918.png">

total size of Mapbox is 11.8 mb

**Comment by JenishKananiLime** [comment]
@OdNairy
